index.html

```diff
@@ -2,172 +2,238 @@
 
 <html lang="en">
 <head>
-  <meta charset="utf-8">
+    <meta charset="utf-8">
 
-  <title>Fun Bus Travel Agency</title>
+    <title>Fun Bus Travel Agency</title>
 
 
-  <link href="css/bootstrap.min.css" rel="stylesheet">
-  <link href="https://fonts.googleapis.com/css?family=Indie+Flower|Roboto" rel="stylesheet">
-  <link href="css/index.css" rel="stylesheet">
+    <link href="css/bootstrap.min.css" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Indie+Flower|Roboto" rel="stylesheet">
+    <link href="css/index.css" rel="stylesheet">
 
-  <!--[if lt IE 9]>
+    <!--[if lt IE 9]>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
-  <![endif]-->
+    <![endif]-->
 </head>
 
 <body>
-  <div class="container-fluid">
+<div class="container-fluid sticky-top">
     <div class="container">
-      <div class="row">
-        <div class="col-3">
-            <h1 class="logo-heading">Fun Bus</h1>
-        </div>
-        <div class="col-9">
-          <nav class="nav">
-            <a class="nav-link active" href="#">Home</a>
-            <a class="nav-link" href="#">About Us</a>
-            <a class="nav-link" href="#">Blog</a>
-            <a class="nav-link" href="#">Contact</a>
-          </nav>
-        </div>
-      </div>
+        <div class="row">
+            <div class="col-3">
+                <h1 class="logo-heading">Fun Bus</h1>
+            </div>
+            <div class="col-9">
+                <nav class="nav justify-content-end mt-3">
```

Comment:
Your nav looks great! One thing I noticed is that your nav links aren't exactly aligned in the center vertically. Just something you can fix with a quick margin-top.